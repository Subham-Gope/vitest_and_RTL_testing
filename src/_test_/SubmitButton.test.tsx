// import { describe, test, expect } from "vitest";
// import { render, screen } from "@testing-library/react";
// import SubmitButton from "../components/SubmitButton";

// describe("testing the SubmitButton.tsx component", () => {
//   test("find button by role", () => {
//     const { container } = render(<SubmitButton />);

//     expect(container).toMatchSnapshot();

//     const btn = screen.getByRole("button");
//     expect(btn).toBeInTheDocument();
//   });
// });
