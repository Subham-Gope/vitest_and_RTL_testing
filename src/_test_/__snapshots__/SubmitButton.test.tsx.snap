// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testing the SubmitButton.tsx component > find button by role 1`] = `
<div>
  <button>
    Submit
  </button>
</div>
`;
